docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Staging release notes
 
+## November 2023
+
+**09 November 2023**
+
+Frontend version: 1.0.0-20231102.1605 | Backend version: 1.71.2-20231108.1413
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** |  Automated self termination and termination requests by Public officers | In our [service request form](https://go.gov.sg/seed-techpass-support), users can now automate the account termination process for employees who have left the organization. Previously handled manually by L1 and L2 engineers, this process can now be automated to save time. Automation is achieved by calling our Automation API, where the following steps are executed: <br>- Retrieve user information using the user's UPN.<br>- Utilise the `Update user status` endpoint (exclusive to our automation API) to terminate the user's account. <br><br><b>Note</b>: Automation applies only to cases initiated by a Public officer or when the user terminates their account not offboarded to SEED. |
+| **Enhancement** | Removed unused fields from `List User API` response in Automation API | We have removed unused fields: `lastSignInAt`, `lastInteractiveSignInAt`, `lastNonInteractiveSignInAt`, and `seedDevices`. These fields, which were previously included in the API documentation, were found to be unused and have been removed to align with the actual response. |
+| **Enhancement** | Webhook UI enhancements |  Webhooks now display the source application (TechPass or TechBiz), providing clarity to new tenant administrators. A new indicator has been added to highlight CAM required webhook events, specifically for `user-updated` and `user-deleted events`. |
```

Comment:
Better to split the 2 Webhook UI enhancements into 2 rows.

Reply:
oki